cargo-dist/src/tasks.rs

```diff
@@ -1286,11 +1293,11 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         let unix_archive = unix_archive.unwrap_or(ZipStyle::Tar(CompressionImpl::Xzip));
         let checksum = checksum.unwrap_or(ChecksumStyle::Sha256);
 
-        let package_cdylibs = package_cdylibs.unwrap_or(false);
-        let install_cdylibs = if !package_cdylibs {
-            false
+        let package_libraries = package_libraries.clone().unwrap_or(vec![]);
+        let install_libraries = if package_libraries.is_empty() {
+            vec![]
         } else {
-            install_cdylibs.unwrap_or(false)
+            install_libraries.clone().unwrap_or(vec![])
```

Comment:
technically this can be `unwrap_or_default`

Reply:
Surprised clippy didn't yell at me for this.